techniques/system/common/1.0/promises.st

```diff
@@ -124,7 +124,6 @@ bundle common va
       "ncf_local_inputs_20" slist => splitstring(execresult("${list_compatible_inputs} ${sys.cf_version} ${ncf_path}/local 20_cfe_basics", "useshell"), "\n", 10000);
       "ncf_local_inputs_30" slist => splitstring(execresult("${list_compatible_inputs} ${sys.cf_version} ${ncf_path}/local 30_generic_methods", "useshell"), "\n", 10000);
       "ncf_local_inputs_40" slist => splitstring(execresult("${list_compatible_inputs} ${sys.cf_version} ${ncf_path}/local 40_it_ops_knowledge", "useshell"), "\n", 10000);
-      "ncf_local_inputs_50" slist => splitstring(execresult("${list_compatible_inputs} ${sys.cf_version} ${ncf_path}/local 50_techniques", "useshell"), "\n", 10000);
```

Comment:
I understand this change at the moment, because we're all working on it, but I know I won't in a few months time.

Could you replace this deleted line with "# Files from 50_techniques are automatically included by Rudder's metadata.xml mechanism" or something similar here please?